src/editor/components/AddBlock/AddBlock.scss

```diff
@@ -9,6 +9,13 @@ $block: '.#{$ns}add-block';
     $buttonIconSize: 16px;
     $buttonWidth: 76px;
     $buttonHeight: 40px;
+    $zIndex: 110;
```

Comment:
up to you, but if we use this variable one time, maybe we dont need this variable ?